### PR TITLE
Fix errors/bugs in VS2015 build instructions

### PR DESCRIPTION
**Pull request by grasmanek94**

### CODE REVIEW

File: docs/building-msvc.md [hunk 1]
```diff
@@ -196,7 +197,7 @@ While still running the VS2015 x86 Native Tools Command Prompt, go to **D:\\msys
 
 <!-- -->
 
-    git clone git://code.qt.io/qt/qt5.git qt5_6_2
+    git clone https://code.qt.io/qt/qt5.git qt5_6_2
```
Comment (stek29) [inline docs/building-msvc.md]: See #3200

Comment (john-preston) [inline docs/building-msvc.md]: I got this git: link from some Qt manual. Why is that important?

Comment (grasmanek94) [inline docs/building-msvc.md]: because the git link is outdated and doesn't work, http too.

Comment (stek29) [inline docs/building-msvc.md]: @grasmanek94 It is outdated and it's insecure, but it does work.

Comment (grasmanek94) [inline docs/building-msvc.md]: Hmm okay it works again, just tested it now, but it didn't work yesterday for some unknown reason.

**Pull request closed without merging**